hosting/locale/de/LC_MESSAGES/django.po

```diff
@@ -204,36 +205,118 @@ msgstr "CHF/Monat"
 msgid "Start VM"
 msgstr "VM jetzt starten"
 
+#: templates/hosting/emails/new_booked_vm.html:99
+#: templates/hosting/emails/vm_charged.html:99
+#, python-format
+msgid " Your New VM %(vm.name)s at Data Center Light "
+msgstr "Ihre neue VM%(vm.name)s bei Data Center Light "
```

Comment:
@deaspo 208-673 this part is added by you? It looks wrong. (It's in "Ihre" form, which we don't use in dcl.) May I ask where you got the text?